docs/versioned-plugins/include/6.x/plugin_header.asciidoc

```diff
@@ -1,9 +1,11 @@
 ifeval::["{versioned_docs}"!="true"]
+[subs="attributes"]
```

Comment:
Note that this file is a manual copy of https://github.com/elastic/logstash/blob/master/docs/include/plugin_header.asciidoc. Make sure you also apply the changes to the file in the logstash repo. Someday the plugin script will copy the file automatically (gotta dream big), but for now, we have to keep the file in sync manually.